tbx/static_src/sass/components/_rich-text.scss

```diff
@@ -1,11 +1,42 @@
 @use "config" as *;
 
 .rich-text {
+    @include font-size(body);
+
+    h3 {
+        @include font-size(heading-three);
+        margin-bottom: $spacer-small;
+    }
+
+    h4 {
+        @include font-size(heading-four);
+        margin-bottom: $spacer-small;
+    }
+
     ul {
         list-style: inside disc;
+        margin-bottom: $spacer-small;
     }
 
     ol {
         list-style: inside decimal;
+        margin-bottom: $spacer-small;
+    }
+
+    a {
+        @include link-styles(var(--color--text), var(--color--text));
+    }
+
+    p {
+        margin-bottom: $spacer-small;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    b,
+    strong {
+        font-weight: $weight--semibold;
```

Comment:
Re-assigning bold tags to use `600` font-weight (and not `700` which we don't include)